### ISSUE TITLE
It is need init CC.IOCQES and CC.IOSQES

### ISSUE THREAD

**Issue by nerudo** [coltonshane/WAVE-Vivado#1]
Some drives fail to create Completion/Submission queues when CC.IOCQES and CC.IOSQES parameters are left in the default state.
It is need to write
`	*regCC |= 0x00460000;`
before enabling controller in the function nvmeInitController()

https://github.com/coltonshane/WAVE-Vivado/blob/56b24427ed793903d3c81d3b9ecff209afae9f29/base_cmd.vitis/WAVE_TestSSD/src/nvme.c#L401

**Comment by coltonshane** [comment]
Yes, sorry, I noticed this also while testing some other SSDs. This has been addressed in a newer version of this repo: [https://github.com/coltonshane/SSD_Test](https://github.com/coltonshane/SSD_Test). (I moved it to Vivado 2021.1, added project creations scripts, and addressed this issue.) I will add a link to the description of this repo to forward to the new one. Thanks!
